src/App.vue

```diff
@@ -521,6 +521,12 @@ function keyboardNoteOn(index: number) {
 // === Typing keyboard state ===
 function windowKeydownOrUp(event: KeyboardEvent | MouseEvent) {
   const target = event.target;
+  // Keep typing activated while adjusting sliders
+  if (target instanceof HTMLInputElement && target.type === "range") {
```

Comment:
We do have a radio button group as well, so it might make sense to make this 
``` 
if (target instanceof HTMLInputElement && ["range", "radio"].includes(target.type)) {
``` 

I also have an idea for the other inputs, which I'll put on the associated issue. But I think this could go in now, either as you have it or with my suggested modification.

Reply:
Good catch!